src/DecisionTree.jl

```diff
@@ -104,7 +109,7 @@ function get_encoding(classes_seen)
 end
 
 MLJBase.fitted_params(::DecisionTreeClassifier, fitresult) =
-    (tree_or_leaf=fitresult[1], encoding=get_encoding(fitresult[2]))
+    (tree=fitresult[1], encoding=get_encoding(fitresult[2]))
 
```

Comment:
Actually, DecisionTree has two distinct types, one for leaves and one for trees (which is unfortunate). If only a stump is has a different type (leaf). But I agree, the label is a little wonky.

Reply:
Yeah I thought it didn't matter... would you prefer if that was reverted?

Reply:
Nah. I've just checked the registry again for good measure. There is a problem with `Resample` which is not really a public model anyway. I've fixed it (most likely) in MLJBase and the registry update can wait till next time. 

Really good to have this new model finally added. Thanks!